data/src/main/java/me/androidbox/data/mapper/ExtensionMapper.kt

```diff
@@ -13,12 +32,12 @@ fun EventEntity.toEvent(): Event {
         remindAt = this.remindAt,
         eventCreatorId = this.eventCreatorId,
         isUserEventCreator = this.isUserEventCreator,
-        attendees = this.attendees,
-        photos = this.photos
+        attendees = attendeeConverter.fromJson(this.attendees),
+        photos = this.photos.fromJson()
     )
 }
```

Comment:
Since you're using type converters, you don't need to call these manually. Room does that for you. So if you define a type converter which teaches Room how to save a `List<Attendee>`, you can make this list directly part of your entity class and don't need to make it a String